### PR TITLE
Added three scenarios and refactored the code

### PR DESCRIPTION
**Pull request by prsathmrc**
PIL-1362, Added PillarId in header for External-Test-Stub, Pillar2-Stub, Pillar2(backend).
Added three scenarios and refactored the code.

### CODE REVIEW

File: src/test/resources/features/UKTRScenarios.feature [hunk 1]
```diff
@@ -3,21 +3,35 @@ Feature: UKTR Scenarios
...
   Scenario Outline: Verify that the Group has not enrolled
     Given I have generated a bearer token for an <UserType> and <Enrolment>
-    And I make API call to PLR UKTR
+    And I make API call to <RequestUrl> and <Endpoint> and <PLRID>
     Then I verify the response code is <StatusCode>
     Examples:
-      | UserType     | Enrolment         | StatusCode |
-      | Organisation | with enrolment    | 201        |
-      | Organisation | without enrolment | 401        |
+      | UserType     | Enrolment         | StatusCode | RequestUrl     | Endpoint      |
```
Comment (ridow-hmrc) [inline src/test/resources/features/UKTRScenarios.feature:9]: where is the PLRID defined here?

File: src/test/resources/jsonSchema/uktrSchema/Response/UKTR_201.json [hunk 2]
```diff
@@ -1,41 +1,19 @@
...
-          "type": "string"
-        }
-      },
-      "required": [
-        "chargeReference",
-        "formBundleNumber",
-        "processingDate"
-      ],
-      "title": "Success"
+      "title": "Welcome7"
```
Comment (ridow-hmrc) [inline src/test/resources/jsonSchema/uktrSchema/Response/UKTR_201.json:16]: why is this called `Welcome7`? Also, I don't think this is the correct response payload

Comment (ridow-hmrc) [reply src/test/resources/jsonSchema/uktrSchema/Response/UKTR_201.json:16]: or maybe it isn't correct based on the new branch that I have. I think you'll have to change this quite soon

File: src/test/resources/features/SubmitUKTR.feature [hunk 3]
```diff
@@ -1,4 +1,3 @@
-@apiTests
```
Comment (ridow-hmrc) [inline src/test/resources/features/SubmitUKTR.feature:1]: is this file still needed? If not, please delete it

File: src/test/scala/uk/gov/hmrc/api/helpers/IdentifyGroupHelper.scala [hunk 4]
```diff
@@ -27,18 +27,23 @@ import java.nio.charset.StandardCharsets
 
 class IdentifyGroupHelper {
```
Comment (ridow-hmrc) [inline src/test/scala/uk/gov/hmrc/api/helpers/IdentifyGroupHelper.scala:28]: This isn't a good name. I would suggest HttpClientHelper or something similar

Comment (prsathmrc) [reply src/test/scala/uk/gov/hmrc/api/helpers/IdentifyGroupHelper.scala:28]: As discussed there are many classes that can be refactored later as this is not related to 1362

File: src/test/resources/features/UKTRScenarios.feature (hunk 1 above)
Comment (prsathmrc) [reply src/test/resources/features/UKTRScenarios.feature:9]: This is not related to pil-1362, In order to work with the new URL I have just parameterized to pass the scenario.

Comment (prsathmrc) [reply src/test/resources/features/UKTRScenarios.feature:9]: Created a new ticket for refactoring the code -PIL-1516

File: src/test/resources/features/SubmitUKTR.feature (hunk 3 above)
Comment (prsathmrc) [reply src/test/resources/features/SubmitUKTR.feature:1]: Created ticket -PIL-1516 to refactor

**Review by ridow-hmrc — APPROVED** [review]
